placideirandora/epic-mail
# #164689039 tested the challenge 3 api endpoints

#### What does this PR do?
It tests for all API endpoints of challenge 3 and fixes some issues.

#### Description of Task to be done?
1. It creates tests for testing user endpoints.
2. It creates tests for testing email endpoints.
3. It creates tests for testing group endpoints.

#### How should this be manually tested?
1. Clone this repository.
2. Download and Install **PostreSQL** database system.
3. Create a database called '**epic-mail**'.
4. Open it with your favorite code editor, but I recommend **VS Code** and **Atom**.
5. Run **npm install** for installing the required dependencies.
6. Run **npm test** to check if the tests pass with 90% coverage.

#### What is the pivotal tracker story?
[Testing challenge endpoints ](https://www.pivotaltracker.com/n/projects/2314363/stories/164689039)
